devicetypes/smartthings/zigbee-range-extender.src/zigbee-range-extender.groovy

```diff
@@ -0,0 +1,61 @@
+/**
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Zigbee Range Extender", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.networking") {
```

Comment:
is there a generic metadata we need to put here or is the app smart enough to just show us a blank device?

Reply:
Well, device page looks fine, there's listed status, but device's tile on device list shows only "Checking status". We can deal with it with metadata or that's job for plugin guys?

Reply:
It'd have to be metadata. You can point it to the Z-Wave extender's metadata: `"SmartThings-smartthings-Z-Wave_Range_extender"`

Reply:
Unfortunately, there's still only "Checking status..."

Reply:
@dkirker do you know how the Aeon Range Extender metadata works?